xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/SignerListSetIT.java

```diff
@@ -51,11 +52,12 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class SignerListSetIT extends AbstractIT {
+public class SignerListSetIT extends BaseIT {
 
   protected final ObjectMapper objectMapper = ObjectMapperFactory.create();
   protected final XrplBinaryCodec binaryCodec = new XrplBinaryCodec();
   protected final KeyPairService keyPairService = DefaultKeyPairService.getInstance();
+  private final XrplClient xrplClient = xrplClient();
```

Comment:
We can pull all of these into `BaseIT` as a field

Reply:
Argh. This was dumb.